### DISCUSSION TITLE
Add dedicated "New tab" button like in Chrome

### DISCUSSION THREAD

**Discussion by Leopold702** [bromite/bromite#2275] [Ideas]
Google Chrome has an option to add a very convenient "New tab" button to the toolbar.

![Screenshot_20220820-224727](https://user-images.githubusercontent.com/64221830/185763914-abd37c0b-29a8-4152-99d4-a7ced13a8be9.png)

It would be nice to have in in Bromite as well.

**Comment by csagan5** [comment]
I do not know why it is not available in v104; @uazo perhaps something related to the segmentation platform?

**Reply by uazo** [reply]
> I do not know why it is not available in v104

it is due to tflite being dropped from the build. 
by heart, the function was activated if the "ia" allowed it, after analyzing the buttons pressed by the user.

**Comment by uazo** [comment]
I checked, it can be activated by the flag `chrome://flags/#adaptive-button-in-top-toolbar-customization`

**Reply by Leopold702** [reply]
Yes, it works. Thank you!

**Reply by Leopold702** [reply]
Unfortunately, this flag was removed in the version 108. It can be temporary unexpired, but probably will be removed for good at some point.

**Reply by Erza** [reply]
Same problem here, the flag is gone on the latest Bromite version.

**Reply by csagan5** [reply]
I'll try to restore it, but it's not a privacy-related feature.
